app/frames/waitlists/[slug]/join/route.tsx

```diff
@@ -320,10 +324,50 @@ const frameHandler = frames(async (ctx) => {
         };
       }
     }
+    if (requiredBuilderScore) {
+      const talentPassportProfile = await getTalentPassportByWallet(
+        ctx.message.requesterVerifiedAddresses[0] ??
```

Comment:
Shouldn't we use userToAdd.address?

Reply:
Oh god yes, indeed.
I was tired and my brain farted lol